catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/operation/TestDorisTableOperations.java

```diff
@@ -439,7 +445,19 @@ public void testCreateTableWithPartition() {
 
     // create table with range partition
     String rangePartitionTableName = GravitinoITUtils.genRandomName("range_partition_table");
-    Transform[] rangePartition = new Transform[] {Transforms.range(new String[] {col4.name()})};
+    LocalDate today = LocalDate.now();
+    LocalDate tomorrow = today.plusDays(1);
+    Literals.LiteralImpl<LocalDate> todayLiteral = Literals.dateLiteral(today);
+    Literals.LiteralImpl<LocalDate> tomorrowLiteral = Literals.dateLiteral(tomorrow);
+    RangePartition rangePartition1 = Partitions.range("p1", todayLiteral, Literals.NULL, null);
+    RangePartition rangePartition2 = Partitions.range("p2", tomorrowLiteral, todayLiteral, null);
+    RangePartition rangePartition3 = Partitions.range("p3", Literals.NULL, tomorrowLiteral, null);
+    Transform[] rangePartition =
```

Comment:
The Transform needs to be verified after creation

Reply:
I mean we should verify that the pre-assigned partitions were successfully created in Doris